source/connect.txt

```diff
@@ -28,6 +28,182 @@ Connect to MongoDB
 ..   /connect/server-selection
 ..   /connect/stable-api
 
-.. TODO, the rest of this root page will be filled out in DOCSP-43745, after
-.. all the child pages have been created. 
-.. This page is acting as a stub for now, so that child pages can be made.
\ No newline at end of file
+Overview
+--------
+
+This page contains code examples that show how to use the
+{+driver-short+} to connect your application to MongoDB by specifying
+various settings. 
+
+.. tip:: Connection Options
+
+   To learn more about the connection options on this page, see the link
+   provided in each section.
+
+To use a connection example from this page, copy the code example into the
+:ref:`sample application <c-connect-sample>` or your own application.
+Be sure to replace all placeholders in the code examples, such as
+``<hostname>``, with the relevant values for your MongoDB deployment.
+
+.. _c-connect-sample:
+
+.. include:: /includes/usage-examples/sample-app-intro.rst
+
+.. literalinclude:: /includes/usage-examples/connect-sample-app.c
+   :language: c
+   :copyable: true
+   :linenos:
+   :emphasize-lines: 12-14
+
+Connection
+----------
+
+The following sections describe how to connect to different targets,
+such as a local instance of MongoDB or a cloud-hosted instance on Atlas.
+
+Local Deployment
+~~~~~~~~~~~~~~~~
+
+The following code shows the connection string to connect to a local
+instance of MongoDB:
+
+.. code-block:: c
+
+   mongoc_uri_t *uri = mongoc_uri_new ("mongodb://localhost:27017"); 
+   mongoc_client_t *client = mongoc_client_new_from_uri (uri);
+
+   // Do database work here
+
+   mongoc_client_destroy (client);
+   mongoc_uri_destroy (uri);
+
+Atlas
+~~~~~
+
+The following code shows the connection string to connect to a
+deployment hosted on Atlas:
+
+.. code-block:: c
+
+   char *uri = "mongodb+srv://<db_username>:<db_password>@<hostname:port>/?<options>";
+   mongoc_client_t *client = mongoc_client_new (uri);
+
+   // Do database work here
+
+   mongoc_client_destroy (client);
+
+Replica Set
+~~~~~~~~~~~
+
+The following code shows the connection string to connect to a
+replica set:
+
+.. code-block:: c
+   
+   char *uri = "mongodb://<replica-set-member>:<port>/?replicaSet=<replica-set-name>";
+   mongoc_client_t *client = mongoc_client_new (uri);
```

Comment:
Nit: we use underlines for the placeholders elsewhere in this page, I think we should stick to that.

```suggestion
   char *uri = "mongodb://<replica_set_member>:<port>/?replicaSet=<replica_set_name>";
   mongoc_client_t *client = mongoc_client_new (uri);
```